wkcuber/api/view.py

```diff
@@ -41,6 +42,7 @@ def __init__(
         self._is_bounded = is_bounded
         self.read_only = read_only
         self._is_opened = False
+        self._mag_view_bbox_at_creation = mag_view_bbox_at_creation
```

Comment:
Please add a comment which explains the reasoning we had about this (we don't want to store a reference to the mag and therefore pass a snapshot of the bbox which can be used for the alignment checks, since it's not super critical when the warning will be (not) emitted incorrectly if the bounding box changes in between).